packages/toolkit/src/query/core/buildInitiate.ts

```diff
@@ -319,6 +320,7 @@ You must add the middleware for RTK-Query to function correctly!`,
           originalArgs: arg,
           queryCacheKey,
           [forceQueryFnSymbol]: forceQueryFn,
+          ...rest
```

Comment:
though it's probably worth moving this spread *before* the other parameters, i think

Reply:
Sure